pkg/features/antrea_features.go

```diff
@@ -81,14 +84,15 @@ var (
 	// To add a new feature, define a key for it above and add it here. The features will be
 	// available throughout Antrea binaries.
 	DefaultAntreaFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
-		AntreaPolicy:       {Default: true, PreRelease: featuregate.Beta},
-		AntreaProxy:        {Default: true, PreRelease: featuregate.Beta},
-		Egress:             {Default: false, PreRelease: featuregate.Alpha},
-		EndpointSlice:      {Default: false, PreRelease: featuregate.Alpha},
-		Traceflow:          {Default: true, PreRelease: featuregate.Beta},
-		FlowExporter:       {Default: false, PreRelease: featuregate.Alpha},
-		NetworkPolicyStats: {Default: true, PreRelease: featuregate.Beta},
-		NodePortLocal:      {Default: false, PreRelease: featuregate.Alpha},
+		AntreaPolicy:              {Default: true, PreRelease: featuregate.Beta},
+		AntreaProxy:               {Default: true, PreRelease: featuregate.Beta},
+		Egress:                    {Default: false, PreRelease: featuregate.Alpha},
+		EndpointSlice:             {Default: false, PreRelease: featuregate.Alpha},
+		Traceflow:                 {Default: true, PreRelease: featuregate.Beta},
+		FlowExporter:              {Default: false, PreRelease: featuregate.Alpha},
+		NetworkPolicyStats:        {Default: true, PreRelease: featuregate.Beta},
+		NodePortLocal:             {Default: false, PreRelease: featuregate.Alpha},
+		SecondaryNetwork:          {Default: true, PreRelease: featuregate.Alpha},
```

Comment:
renamed as SecondaryNetwork (more relevant to state it as network instead of interface, in my opinion - as we do configure secondary network to address CNF use cases with this feature gate)

Reply:
I think this should be `SecondaryNetworks`...